guides/source/getting_started.md

```diff
@@ -194,6 +165,38 @@ of the files and folders that Rails creates by default:
 |.gitignore|This file tells git which files (or patterns) it should ignore. See [GitHub - Ignoring files](https://help.github.com/articles/ignoring-files) for more information about ignoring files.|
 |.rubocop.yml|This file contains the configuration for RuboCop.|
 |.ruby-version|This file contains the default Ruby version.|
+|.devcontainer/|This folder contains the Dev Container configuration|
+
+### Opening the Blog Application in a Dev Container
+
+A Dev Container is a fully functional development environment running in a container
+on your machine. Our new Rails application comes with a Dev Container already
+configured and ready to use.
+
+We will use VSCode to spin up and work with our Dev Container. First we need to open
+our app in VSCode. This can be done by opening VSCode and then opening our application's
+folder, or directly from the command line within our application's directory with:
+
+```bash
+$ code .
+```
+
+When we open the application, VSCode will recognize that a Dev Container is available
+and will prompt us to re-open the application in a Dev Container. Click the prompt
```

Comment:
maybe we can allude to what the prompt is, something like "Folder contains a Dev Container configuration file. Reopen folder to develop in a container ([learn more](https://aka.ms/vscode-remote/docker))."

Reply:
Maybe we should also add a reminder to have the Docker application running?

Reply:
I was thinking to avoid matching the language exactly as VScode could change it. But I will make it a bit more clear.

Reply:
Sounds good!